src/eei.cpp

```diff
@@ -45,6 +46,25 @@ bool exceedsUint128(evmc_uint256be const& value) noexcept
 }
 }  // namespace
 
+bool WasmEngine::benchmarkingEnabled = false;
```

Comment:
I think this should go into the header.

Reply:
That would define the variable multiple times.

Reply:
Why?

Reply:
Because the header is included in multiple compilation units. You will not be able to link it in the end.

Reply:
Maybe there's some confusion here. I do not suggest to move this statement as is, but: https://github.com/ewasm/hera/pull/486/files#r256387031